packages/amplify-graphiql-explorer/scripts/patch.js

```diff
@@ -0,0 +1,74 @@
+const fs = require("fs");
+const path = require("path");
+const nodeModulesPath = path.resolve(`${process.cwd()}/../..`, "node_modules");
```

Comment:
Builds were failing because of an incompatibility with `semantic-ui` and `webpack5`. A patch package was created to bridge the issue (https://github.com/Semantic-Org/Semantic-UI/issues/7073#issuecomment-1016579441). 

This script is a lightly modified version of [that patch package](https://github.com/layershifter/semantic-ui-tools/blob/master/packages/css-patch/src/index.js#L6) modified to work with our lerna setup.